### ISSUE TITLE
Change directory of sort algorithms

### ISSUE THREAD

**Issue by Sumith1896** [pydsa/pydsa#20]
All the sort algorithms currently reside in `pydsa/`, move them to `sorting/`
Inside this we can have all the sorting algorithms in separate files as we have now else we can have them in one file.

**Comment by aktech** [comment]
Yes, that would be better now, since the number of files were less, I had kept them in one file.

**Comment by manu-chroma** [comment]
Hi, looks like there is no progress in PR https://github.com/pydsa/pydsa/pull/82 regarding this issue.

If no one has taken this issue, I can complete this task and fix any failing tests accordingly.
